src/bundle/Controller/ContentViewController.php

```diff
@@ -462,4 +475,15 @@ private function supplyIsLocationBookmarked(ContentView $view): void
 
         $view->addParameters(['location_is_bookmarked' => $locationIsBookmarked]);
     }
+
+    /**
+     * @param \eZ\Publish\Core\MVC\Symfony\View\ContentView $view
+     */
+    private function supplyContentReverseRelations(ContentView $view)
+    {
+        $contentInfo = $view->getLocation()->getContentInfo();
+        $relations = $this->contentService->loadReverseRelations($contentInfo);
+
+        $view->addParameters(['contentHasReverseRelations' => (bool)count($relations)]);
```

Comment:
I think we use undercase for view parameters.

and that could be only me, but i think that count() > 0 is better than casting to bool.